sagemaker-featurestore/feature_store_securely_store_images.ipynb

```diff
@@ -0,0 +1,680 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_show_image(df):\n",
+    "    \"\"\"\n",
+    "    df: dataframe of your data\n",
+    "    \n",
+    "    Return a numpy array of your images that have been reshaped into it's corresponding shape. \n",
+    "    \"\"\"\n",
+    "    import ast\n",
+    "    images = []\n",
+    "    for index, entry in enumerate(np.array(df['img'])):\n",
+    "        entry = entry.strip('b')\n",
+    "        entry = np.array(ast.literal_eval(ast.literal_eval(entry)))\n",
+    "        shape = ast.literal_eval(df['shape'][index])\n",
+    "        entry = entry.reshape(shape[0], shape[1])\n",
+    "        images.append(entry)\n",
+    "    return np.array(images)\n",
+    "images = parse_show_image(dataset)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Below shows the shape of your image data set. \n",
+    "images.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot the images to demonstrate that you can view the images stored in your feature store. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for img in images:\n",
+    "    plt.imshow(img)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Clean Up Resources\n",
+    "Remove the Feature Group that was created. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group.delete()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Next Steps\n",
+    "\n",
+    "In this notebook we covered how to securely store data sets of images in a feature store using KMS key.\n",
+    "\n",
+    "If you are interested in understanding more on how server-side encryption is done with Feature Store, see [Feature Store: Encrypt Data in your Online or Offline Feature Store using KMS key](https://sagemaker-examples.readthedocs.io/en/latest/sagemaker-featurestore/feature_store_kms_key_encryption.html).\n",
+    "\n",
+    "If you are interested in understanding how to do client-side encryption to encrypt your image data set prior to storing it in your feature store, see [Amazon SageMaker Feature Store: Client-side Encryption using AWS Encryption SDK](https://sagemaker-examples.readthedocs.io/en/latest/sagemaker-featurestore/feature_store_client_side_encryption.html). For more information on the AWS Encryption library, see [AWS Encryption SDK library](https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/introduction.html).\n",
```

Comment:
"If you are interested in understanding how to do client-side encryption to encrypt your image dataset prior to storing it in your feature store, see [Amazon SageMaker Feature Store: Client-side Encryption using AWS Encryption SDK](https://sagemaker-examples.readthedocs.io/en/latest/sagemaker-featurestore/feature_store_client_side_encryption.html). For more information on the AWS Encryption library, see [AWS Encryption SDK library](https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/introduction.html).\n",